docs/src/lib/services/onboard.js

```diff
@@ -106,6 +105,8 @@ const intiOnboard = async (theme) => {
     apiKey: 'pk_live_02207D744E81C2BA'
   })
 
+  const frame = frameModule()
+
   return Onboard({
     wallets: [
```

Comment:
Do we want to add Frame to the wallets list? If so prob a little lower in the lineup

Reply:
nm just saw it down there!